content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm.md

```diff
@@ -56,20 +56,118 @@ Example usage:
 </colgroup>
 <tbody>
 
+<tr>
+<td colspan="2">--add-dir-header</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>If true, adds the file directory to the header of the log messages</p></td>
```

Comment:
Hi @tengqm .
Did the upstream code change (post v1.21.0) and now you can generate the "Optional parent flags" for each command?
It seems somewhat repetitive. Do all commands share these common or parent commands (or most commands)?

Reply:
Yes.

It looks like most sub-commands share the same set of parent command options.